MayssaraOmar/assignment4/Counting islands.cpp

```diff
@@ -0,0 +1,125 @@
+#include "test.h"
+#include <iostream>
+#include <vector>
+using namespace std;
+
+static void markIsland(
+	const vector<vector<bool>> &grid,
```

Comment:
Indentation is a bit off here, parameters should be a little more Indented than the code.
Also, having one in the first line may be more clear.